public/ng-views/maps.html

```diff
@@ -629,47 +731,106 @@ <h3 class="noTopMargin noBottomMargin">
           <h6 ng-click="toggleShowVectorIsos()">
             {{ vectorIsos.val*5 | minuteConverter }}
             <i class="fa fa-eye" ng-show="showVectorIsosOn"></i>
             <i class="fa fa-eye-slash" ng-show="!showVectorIsosOn"></i>
           </h6>
-      </span>
+      </span> -->
     </div>
-  </div>
 
-  <h4 ng-click="combosView=!combosView">
-    <span ng-show="!combos.sel">
-      Existing MBTA System
-    </span>
-    <span ng-show="combos.sel">
-      Scenario {{ combos.all[combos.sel].name }}
-      <span ng-show="scenarioCompare">
-        <span ng-show="combos.com">
-          Compared with {{ combos.all[combos.com].name }}
-        </span>
-        <span ng-show="!combos.com">
-          Compared with Baseline MBTA System
-        </span>
-      </span>
-    </span>
-      <i class="fa fa-angle-double-up pull-right" ng-show="!combosView"></i>
-      <i class="fa fa-angle-double-down pull-right" ng-show="combosView"></i>
-  </h4>
+  <h5 style="margin-bottom: -20px"> DESTINATIONS </h4> 
+    <br>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-briefcase fa-fw destinationIcon"></i>
+           <!--  <figcaption class="caption">Jobs</figcaption> -->
+    </label>
+    <label  class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-heartbeat fa-fw destinationIcon"></i>
+          <!-- <figcaption class="caption">Healthcare</figcaption> -->
+    </label>
+    <label  class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+          <i class="fa fa-shopping-cart fa-fw destinationIcon"></i>
+          <!-- <figcaption class="caption">Groceries</figcaption> -->
+    </label>
+    <label  class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+          <i class="fa fa-graduation-cap fa-fw destinationIcon"></i>
+         <!--  <figcaption class="caption">K1-12</figcaption> -->
+    </label>
+  </h5>
+  <br>
+
+  <h5 style="margin-bottom: -20px"> MODE </h4> 
+    <br>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-bus destinationIcon"></i>
+    </label>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-bicycle destinationIcon"></i>
+    </label>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-child destinationIcon"></i>
+    </label>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-ship destinationIcon"></i>
+    </label>
+  </h5>
+</div>
+<!--End of DESTINATION & TIME FILTER --> 
```

Comment:
no need for end comments if everything is correctly formatted (which is why we need to rm those commented out components; makes it super hard to read)